# Message for sending out data read from the msp430
uint8[] data